beniget/beniget.py

```diff
@@ -1,21 +1,29 @@
 from collections import defaultdict
 from contextlib import contextmanager
+import builtins
 import sys
 
-import gast as ast
+import gast
 
 from .ordered_set import ordered_set
 
-class Ancestors(ast.NodeVisitor):
+
+def pkg(node):
+    return sys.modules[type(node).__module__]
```

Comment:
according to `%timeit` this is faster if written as follow:

`ast if s.__class__.__module__ == 'ast' else gast`